setup.py

```diff
@@ -16,12 +16,10 @@
 tests_requires = ["pytest~=4.0", "pytest-cov~=2.0"]
 
 install_requires = [
-    "requests>=2.20",  # for common use in custom actions code
     "ConfigArgParse>=0.14",
     "coloredlogs~=10.0",
-    "flask~=1.0",
-    "flask_cors~=3.0",
-    "gevent>=1.4",
+    "sanic~=19.6",
+    "sanic-cors==0.9.9.post1",
```

Comment:
how about going straight to `0.9.9.post3`?

Reply:
same as the sanic package